Doc/reference/executionmodel.rst

```diff
@@ -163,14 +166,65 @@ These references follow the normal rules for name resolution with an exception
 that unbound local variables are looked up in the global namespace.
 The namespace of the class definition becomes the attribute dictionary of
 the class. The scope of names defined in a class block is limited to the
-class block; it does not extend to the code blocks of methods -- this includes
+class block; it does not extend to the code blocks of methods. This includes
 comprehensions and generator expressions since they are implemented using a
-function scope.  This means that the following will fail::
+function scope, but it does not include :ref:`annotation scopes <annotation-scopes>`,
```

Comment:
Let's leave out the part about "implemented using a function scope," since it's no longer true for comprehensions, and isn't critical to the point of this section anyway?